experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -20,12 +20,24 @@
 CACHE_HOST="http://bazel-cache.default:8080"
 
 make_bazel_rc () {
+    # this is the default for recent releases but we set it explicitly
+    # since this is the only hash our cache supports
     echo "startup --host_jvm_args=-Dbazel.DigestFunction=sha256" >> $1
+    # caching args for build
+    # - use remote strategy for all the things
     echo "build --spawn_strategy=remote" >> $1
```

Comment:
does this mean everything is executing remotely?

Reply:
no, they definitely execute locally, but this controls how sandboxing etc works as well AFAICT? this flag is from the buchgr/bazel-remote http/rest caching docs.
I'm actually about to update this to the args specified here:
https://docs.bazel.build/versions/master/remote-caching.html#read-from-and-write-to-the-remote-cache